source/blood/src/menu.cpp

```diff
@@ -569,8 +570,9 @@ const char *pzMusicDeviceStrings[] = {
 };
 
 CGameMenuItemTitle itemOptionsSoundTitle("SOUND SETUP", 1, 160, 20, 2038);
-CGameMenuItemZBool itemOptionsSoundSoundToggle("SOUND:", 3, 66, 60, 180, false, UpdateSoundToggle, NULL, NULL);
-CGameMenuItemZBool itemOptionsSoundMusicToggle("MUSIC:", 3, 66, 70, 180, false, UpdateMusicToggle, NULL, NULL);
+CGameMenuItemZBool itemOptionsSoundSoundToggle("SOUND:", 3, 66, 50, 180, false, UpdateSoundToggle, NULL, NULL);
+CGameMenuItemZBool itemOptionsSoundMusicToggle("MUSIC:", 3, 66, 60, 180, false, UpdateMusicToggle, NULL, NULL);
+CGameMenuItemZBool itemOptionsSoundMusicRestartsOnLoadToggle("MUSIC RESTARTS:", 3, 66, 70, 180, false, UpdateMusicRestartsOnLoadToggle, NULL, NULL);
```

Comment:
I don't think this option is worth exposing to menu

Reply:
If you are sure that you want to change by default the original behavior then I agree, the console command would be enough for 99.99% of the players.